CRM/Core/DAO.php

```diff
@@ -2049,9 +2049,8 @@ public static function createTestObject(
       foreach ($fields as $fieldName => $fieldDef) {
         $dbName = $fieldDef['name'];
         $FKClassName = $fieldDef['FKClassName'] ?? NULL;
-        $required = $fieldDef['required'] ?? NULL;
```

Comment:
Unused var

Reply:
But it says "required" (grin).

Seriously these all look ok to me.